cogs/misc.py

```diff
@@ -26,6 +26,15 @@ def f_to_c(f: float) -> float:
 class Miscellaneous(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.someone_emotes = [u'\u0CA0_\u0CA0',
```

Comment:
This should be moved to `util/emoji_chars.py`  
Also, use `\N{EMOJI_NAME}` syntax instead of using `u/code`

Reply:
I'll move it, but these are the unicode chars (Kanji maybe?) which are used in emotes like Lenny, so not emojis.

Reply:
That syntax is for unicode characters. I said emojis because that's what discord calls them

![image](https://user-images.githubusercontent.com/47357913/81124123-145bc100-8f4e-11ea-96e3-4d370175c7c5.png)

Reply:
I *could* switch to that syntax, but that's gonna give some really long strings (up to over 200 chars) so I'd rather not unless you definitely require that.

Reply:
Then please just put the unicode characters directly. These codes are an unreadable mess